### ISSUE TITLE
Not stable on Windows

### ISSUE THREAD

**Issue by seifibrahim32** [dart-lang/jnigen#195]
.

**Comment by HosseinYousefi** [comment]
Didn't it work in #194?

**Comment by seifibrahim32** [comment]
I need to work it from the Flutter code not the command , command returns the output,


On Android returns when I run from Flutter.


on Windows shows dll files issues

**Comment by seifibrahim32** [comment]
Need more information mr Hossein?

**Comment by seifibrahim32** [comment]
![Uploading IMG_20230305_190837_884.jpg …]()

![Uploading IMG_20230305_190908_065.jpg …]()

**Comment by seifibrahim32** [comment]
![scrree](https://user-images.githubusercontent.com/58334300/222976381-df763ea5-3497-4e2d-9275-7ba2ffe86115.PNG)

**Comment by seifibrahim32** [comment]
@HosseinYousefi

**Comment by seifibrahim32** [comment]
Any updates mr Hossein?

**Comment by HosseinYousefi** [comment]
Hi Seif,

You don't need to write multiple comments and tag me in each one of them! In #194, you have been writing every step in a separate comment. This is rather distracting.

Instead, I suggest you to thoroughly examine the issue yourself, and provide the necessary means for reproduction on my end. I can't reproduce the issue based on an image, right? So you'd have to provide me with a minimal example of a failing project + the exact steps you took.

**I'd rather have all these information in the single top comment of the issue, so I can act on it.**

I will give you updates about the issue when I have them, there is no need for additional comments!

**Comment by seifibrahim32** [comment]
OK mr Hossein , my apologies for this , after this point, any comments will be in single post 🙏.

I will try to solve this issue with myself but, I will tell you what I did.

1- I installed client.dll and pasted the file on Windows System32 folder.

2. I closed the device so the memory re-reads the DLL files.

3. I opened the Android Studio and ran the Flutter Windows.

[ for Jni.spawn() it wrote it at void main().]

5. Closed it and the issue is still as it is.


[Here I talk about Windows only],


Second point and It is the Android part,


It works perfectly on the Android but my point is call I use Android HTTPUrlConnection or RetroFit or Volley library for HTTP requests instead of java.net.http which Android documentation dont support it.?
